origami.json

```
@@ -16,14 +16,14 @@
 		"js": "demos/src/demo.js",
 		"documentClasses": "",
 		"dependencies": [
-			"@financial-times/o-table@9.0.0-0",
-			"@financial-times/o-syntax-highlight@4.0.0-0",
-			"@financial-times/o-tabs@6.0.0-0",
-			"@financial-times/o-header-services@5.0.0-0",
-			"@financial-times/o-footer-services@4.0.0-0",
-			"@financial-times/o-buttons@7.0.0-0",
-			"@financial-times/o-forms@9.0.0-1",
-			"@financial-times/o-normalise@3.0.0-0"
+			"o-table@9.0.0-0",
+			"o-syntax-highlight@4.0.0-0",
+			"o-tabs@6.0.0-0",
+			"o-header-services@5.0.0-0",
+			"o-footer-services@4.0.0-0",
+			"o-buttons@7.0.0-0",
+			"o-forms@9.0.0-1",
+			"o-normalise@3.0.0-0"
```

The demos can use the stable version instead of the prereleases
```suggestion
			"o-table@9.0.0",
			"o-syntax-highlight@4.0.0",
			"o-tabs@6.0.0",
			"o-header-services@5.0.0",
			"o-footer-services@4.0.0",
			"o-buttons@7.0.0",
			"o-forms@9.0.0",
			"o-normalise@3.0.0"
```

---

not all of these have had a release yet

---

(o-forms, o-{header,footer}-services)